SwiftAdvanceExercise.playground/Pages/Enumeration_3.6.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,13 @@
+//MARK:  Write a swift program using enumerations to get all cases as an array with use of CaseIterable(Optional) and without use of CaseIterable(Mandatory).
+
+enum Colors: Int, CaseIterable {
```

Comment:
if you are using CaseIterable then, we don't need to have 
static var allCases
implement the non CaseIterable separately